web/app/routes/mainRoutes.js

```diff
@@ -115,4 +116,14 @@ router.delete("/drivingschools/:schoolId/instructors/:userId", function(req, res
 	res.json({});
 });
 
+// GET the driving regulations for a specified state
+rouer.get("/stateregulations/:state", function(req, res) {
+	stateRegsCtrl.getStateRegs(req.params.state, (sateRegs)=>{
```

Comment:
`sateregs` is not spelled correctly, this won't run because `stateregs` below is undefined